rest/blip_client_test.go

```diff
@@ -34,16 +34,31 @@ type BlipTesterClientOpts struct {
 	Channels               []string
 	SendRevocations        bool
 	SupportedBLIPProtocols []string
+	Collections            []string
 
 	// a deltaSrc rev ID for which to reject a delta
-	rejectDeltasForSrcRev string //nolint: structcheck // false structcheck positive due to https://github.com/golangci/golangci-lint/issues/826
+	rejectDeltasForSrcRev string // nolint: structcheck // false structcheck positive due to https://github.com/golangci/golangci-lint/issues/826
```

Comment:
That comment shouldn't have the space before `nolint` - from https://golangci-lint.run/usage/false-positives/#nolint-directive:

> Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.

Reply:
Hrm pretty sure I didn't change that... I think it might be goimports... Wonder if I can add an exception to avoid it doing that somehow

Reply:
Right okay. Changed to //nolint:structcheck as discussed in slack.